src/meidocument.cpp

```diff
@@ -100,17 +100,17 @@ MeiElement* mei::MeiDocument::getElementById(string id) {
     }
 }
 
-vector<MeiElement*> mei::MeiDocument::getElementsByName(string name) {
+vector<MeiElement*> mei::MeiDocument::getElementsByName(string name) const {
```

Comment:
Here and in other methods: Passing string arguments by const reference should be preferred, i.e. use `const std::string &name`.